generic/tests/netperf.py

```diff
@@ -291,51 +298,56 @@ def _pin_vm_threads(vm, node):
             password = params_tmp["password"]
             username = params_tmp["username"]
             env_setup(i, ip_dict[i], username, shell_port, password)
+        elif params_tmp.get("os_type") == "windows":
+            windows_disable_firewall = params.get("windows_disable_firewall", "")
```

Comment:
If you don't need default value, you can directly write params.get("windows_disable_firewall")

Reply:
windows_disable_firewall could be empty, it depends on requirement.

Reply:
@quanwenli the definition of params.get() is `def get(self, key, default=None)`, it has default value `None`, so no need to specify it if you don't need one.

Reply:
Yes, I will fix it.